pkg/kubelet/dockertools/kube_docker_client.go

```diff
@@ -87,10 +87,21 @@ func newKubeDockerClient(dockerClient *dockerapi.Client, requestTimeout time.Dur
 	if requestTimeout == 0 {
 		requestTimeout = defaultTimeout
 	}
-	return &kubeDockerClient{
+
+	k := &kubeDockerClient{
 		client:  dockerClient,
 		timeout: requestTimeout,
 	}
+	// Notice that this assumes that docker is running before kubelet is started.
+	v, err := k.Version()
+	if err != nil {
```

Comment:
I think should fail here since there is no other place for `dockerClient.UpdateClientVersion`.